web/src/components/Verdict/FinalDecision.tsx

```diff
@@ -84,13 +87,23 @@ interface IFinalDecision {
 
 const FinalDecision: React.FC<IFinalDecision> = ({ arbitrable }) => {
   const { id } = useParams();
+  const { isDisconnected } = useAccount();
   const { data: disputeTemplate } = useDisputeTemplate(id, arbitrable);
   const { data: disputeDetails } = useDisputeDetailsQuery(id);
+  const { wasDrawn, hasVoted, isLoading, isCommitPeriod, isVotingPeriod, commited } = useVotingContext();
   const ruled = disputeDetails?.dispute?.ruled ?? false;
   const navigate = useNavigate();
   const { data: currentRulingArray } = useKlerosCoreCurrentRuling({ args: [BigInt(id ?? 0)], watch: true });
   const currentRuling = Number(currentRulingArray?.[0]);
   const answer = disputeTemplate?.answers?.[currentRuling! - 1];
+  const buttonText = useMemo(() => {
+    if (!wasDrawn || isDisconnected) return "Check how the jury voted";
+    if (disputeDetails?.dispute?.period === "evidence") return "You were drawn";
+    if (isCommitPeriod) return commited ? "You have committed" : "Commit your vote";
+    if (isVotingPeriod && !commited) return "You failed to commit";
+    if (isVotingPeriod && !hasVoted) return "Cast your vote";
+    return hasVoted ? "You have voted" : "You have not appeared for juror duty";
```

Comment:
I think we should re-imagine the texts 🤔  right now I feel like some of them are call to actions and some of them are informative, and that feels confusing.
I would have:
- `Cast your vote` in voting period and not `hiddenVotes`
- `Commit your vote` on commit period
- `Reveal your vote` on voting period with `hiddenVotes`
- leave it with `Check how the jury voted` for other cases

Other texts feel more like information, and I don't think it's intuitive to click them and go to the voting page.

(maybe we could change it to a call to action to check the evidence tab during the first evidence period)

Reply:
Okay let's do that, more simple 👍  

No strong opinion about the CTA for the evidence tab, maybe follow up in a subsequent PR?